globals.json

```diff
@@ -416,7 +416,8 @@
 	"worker": {
 		"importScripts": true,
 		"postMessage": true,
-		"self": true
+		"self": true,
+		"FileReaderSync": true
 	},
 	"node": {
```

Comment:
None of these additions to `node` are actually globals in node (at least as of `0.11.14`).